c7n/filters/offhours.py

```diff
@@ -235,6 +236,8 @@ class Time(Filter):
             'weekends': {'type': 'boolean'},
             'weekends-only': {'type': 'boolean'},
             'opt-out': {'type': 'boolean'},
+            'skip-days': {'type': 'array', 'items': {'type': 'string'}},
```

Comment:
can you do a regex constraint on the string value for skip-days to ensure the format your expecting.

Reply:
examples https://spacetelescope.github.io/understanding-json-schema/reference/regular_expressions.html

Reply:
@kapilt Added!